frontend/server/libs/dao/Courses.dao.php

```diff
@@ -307,9 +307,9 @@ public static function getAssignmentsProgress($course_id, $identity_id) {
         return new Courses($row);
     }
 
-    final public static function getAssignmentByAlias(Courses $course, $assignment_alias) {
+    final public static function getAssignmentByAlias(int $courseId, String $assignmentAlias) {
```

Comment:
Puedes dejar el curso como estaba antes?

```suggestion
    final public static function getAssignmentByAlias(Courses $course, string $assignmentAlias) {
```

también, el tipo de las cadenas es `string` en PHP, no `String`.

Reply:
listo